security/pkg/k8s/tokenreview/k8sauthn.go

```diff
@@ -24,12 +24,14 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"istio.io/istio/pkg/jwt"
+	"istio.io/pkg/env"
 )
 
-const (
-	// The default audience for SDS trustworthy JWT. This is to make sure that the CSR requests
+var (
+	// DefaultAudience specifies a list of audiences for SDS trustworthy JWT. This is to make sure that the CSR requests
 	// contain the JWTs intended for Citadel.
-	DefaultAudience = "istio-ca"
+	DefaultAudience = env.RegisterStringVar("TOKEN_AUDIENCES", "istio-ca",
```

Comment:
nit: add strings.Split(...Get()) here so we don't need to do it every time we call it

Reply:
done, also updated the variable name and comments.